service/node.go

```diff
@@ -934,7 +942,7 @@ func (s *service) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolum
 	err = s.nodeProbe(ctx)
 	if err != nil {
 		Log.Error("nodeProbe failed with error :" + err.Error())
-		return nil, err
+		// return nil, err
```

Comment:
**question:** Why is this return commented out? Is it needed?

Reply:
I will fix that.

Reply:
fixed